### PR TITLE
Zend_Validate_Hostname with latest gTLD

### PR DESCRIPTION
**Pull request by MrGekko**
Update Zend_Validate_Hostname to include the latest gTLD
Version 2015061701, Last Updated Thu Jun 18 07:07:01 2015 UTC

### CODE REVIEW

File: lib/Zend/Validate/Hostname.php [hunk 1]
```diff
@@ -565,14 +902,20 @@ class Zend_Validate_Hostname extends Zend_Validate_Abstract
         'today',
         'tokyo',
         'tools',
+        'top',
+        'toray',
+        'toshiba',
+        'tours',
         'town',
         'toys',
-        'tp',
```
Comment (Flyingmana) [inline lib/Zend/Validate/Hostname.php]: for the case someone wants to know why it got removed
source: https://en.wikipedia.org/wiki/.tp

**Comment by Flyingmana** [comment]
:+1:

**Comment by LeeSaferite** [comment]
:+1:

**Comment by alexkirsch** [comment]
+1

**Pull request merged**